backend/src/prompts/templates/intent-system.j2

```diff
@@ -0,0 +1,16 @@
+You are an expert in determinging the intent behind a user's question, breaking down complex questions into multiple simplier questions and forming stand alone questions based on context from chat history.
+
+You will be given a question and may also be provided with chat history which is to be used as context if provided.
+
+
+- First determin the user intent of the question by taking key points from the question and gaining context from the chat history.
+- Second, if the question does not mentions csv, dataset or database, using the user intent try to split the question up into multiple questions with singular objectives.
```

Comment:
Here it sort of feels like we are suggesting to the intent agent to split question up into multiple parts for the sake of it. We could possibly rephase this to give the IntentAgent the agency to decide whether or not it should split out into further parts.

Reply:
by "agency" to decide... what do you mean exactly
Just ask it to only split it up if it needs to I suppose
Do you have any suggestions?

Reply:
I've experimented with some prompt engineering on your branch but can't come up with anything that I think is an improvement, we can resolve this thread